rootfs-builder/rootfs.sh

```diff
@@ -239,7 +239,9 @@ if [ -n "${USE_DOCKER}" ] ; then
 		--env KERNEL_MODULES_DIR="${KERNEL_MODULES_DIR}" \
 		--env EXTRA_PKGS="${EXTRA_PKGS}" \
 		--env OSBUILDER_VERSION="${OSBUILDER_VERSION}" \
+        --net host \
 		-v "${script_dir}":"/osbuilder" \
+        --privileged \
```

Comment:
Again, is this option essential? If so, why? I was thinking we could run the container with a number of capabilities `--cap-add`.